Docs: Fix document possible value missed of 45 resources (#19270)
hashicorp/terraform-provider-azurerm: website/docs/r/mssql_managed_instance.html.markdown

* `administrator_login_password` - (Required) The password associated with the `administrator_login` user. Needs to comply with Azure's [Password Policy](https://msdn.microsoft.com/library/ms161959.aspx)

> * `license_type` - (Required) What type of license the Managed Instance will use. Valid values include can be `LicenseIncluded` and `BasePrice`.

Reviewer: ```suggestion
* `license_type` - (Required) What type of license the Managed Instance will use. Possible values are `LicenseIncluded` and `BasePrice`.
```
